src/wazuh_modules/vulnerability_detector/wm_vuln_detector.h

```diff
@@ -1136,6 +1136,13 @@ int wm_vuldet_get_hotfixes(int agent_id, cJSON** requested_items);
  **/
 int wm_vuldet_get_software(int agent_id, bool not_triaged, cJSON** requested_items);
 
+/**
+ * @brief Method to normalize the architecture string to the NVD standard.
+ *
+ * @param architecture The previously allocated architecture string.
```

Comment:
This description would change with the suggestion below. Also, you will have to add the return value.

Reply:
Fixed!